package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "jest"
```

Comment:
could be worth adding a `test:update` here with `jest --updateSnapshot` so you can run `npm run test:update` if you need to update the snapshots (same as we do in Hazman)